modules/op-installing-sbo-operator-in-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+[id="op-installing-sbo-operator-in-web-console_{context}"]
+= Installing the {servicebinding-title} in web console
+
+You can install {servicebinding-title} using the {product-title} OperatorHub. When you install the {servicebinding-title}, the custom resources (CRs) required for the service binding configuration are automatically installed along with the Operator.
+
+[discrete]
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators* -> *OperatorHub*.
+
+. Use the *Filter by keyword* box to search for `{servicebinding-title}` in the catalog. Click the *{servicebinding-title}* tile.
+
+. Read the brief description about the Operator on the *{servicebinding-title}* page. Click *Install*.
+
+. On the *Install Operator* page:
++
+.. Select *All namespaces on the cluster (default)* for the *Installation Mode*. This mode installs the Operator in the default `openshift-operators` namespace, which enables the Operator to watch and be made available to all namespaces in the cluster.
+
+.. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
+
+.. Select an *Update Channel*.
+
+*** By default, the *candidate* channel enables installation of the latest stable and supported release of the {servicebinding-title}.
+*** The *preview* channel enables installation of the latest preview version of the {servicebinding-title}, which may contain features that are not yet available from the *candidate* channel and is not supported.
```

Comment:
`preview` channel is not going to be used anymore, since we adopted the suggested RH convention:

https://olm.operatorframework.io/docs/best-practices/channel-naming/